schedule.json

```
@@ -52,14 +52,24 @@
   },
   "v11": {
     "start": "2018-10-23",
-    "maintenance": "2019-04-01",
-    "end": "2019-06-30"
+    "maintenance": "2019-04-22",
+    "end": "2019-06-01"
   },
   "v12": {
     "start": "2019-04-23",
     "lts": "2019-10-22",
     "maintenance": "2021-04-01",
-    "end": "2022-04-01",
-    "codename": ""
```

do we need to keep the codename here?

```suggestion
 "codename": "TBD"
```